index.d.ts

```diff
@@ -714,3 +714,12 @@ interface DurableObjectNamespace {
 
   get: (id: DurableObjectId) => DurableObjectStub;
 }
+
+interface WebSocket {
+  accept(): void;
+}
+
+declare class WebSocketPair extends Array {
+  0: WebSocket;
+  1: WebSocket;
+}
```

Comment:
Sorry, I see that there's been back and forth on this, but this actually isn't correct. Extending an Array means that this could be possible, which it isn't:

```ts
let [client, server] = new WebSocketPair;
```

The correct solution is to do the equivalent of what [worktop](https://github.com/lukeed/worktop/blob/master/src/ws.d.ts#L4-L11) does, which is this:

```suggestion
interface WebSocketPair {
  new(): {
    0: WebSocket;
    1: WebSocket;
  };
};
```

Here's a [Playground](https://www.typescriptlang.org/play?#code/CYUwxgNghgTiAEkoGdnwOogEYGUD2YA1iAC4AKUAljAIzwgAeJIAdsGgIIwxQCe8AbwBQ8eAAYAXBmz4ipANwj4NKZlwFiJRQF8hQgPT74AARLIAtIwAO4Epe54YQiKXgBtSJVYkaAGnjIIDAAbkE0ALrwALzwLCAA7tLqcuRUtIoGRiQAFpRoyNl4AK4QwPBYCHiEfM6uAuISnt4ATP4qgSFBzfDa0bEJSbKaFNQ0GYbw2SQkVsgShgDmlDlFWAB0YHgAtvoQRcQgwPrxjoQkeFb6WBB4WPpbKMww+sgwYMfIa8BrZgDEADIAFnM-xoNCElBYTwAZlAwAg1ENSCMYN1hKJRHF4gAKACUUnRGKJkkGGgUSiJohUpJSiiJ2h04yMpgs1ls9hgjlqJHcTShAGZ-B1QjB+ZEYliacM0s0mfAcnl4IrkCRKBAIPAqjUXDz6pI+SRAW0JMKgoCen1JYiyalRvIgA) to see the differences.